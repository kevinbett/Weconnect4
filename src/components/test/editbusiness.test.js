import EditBusinessForm from "../EditBusinessBody/index"
import { shallow } from "enzyme"

describe('login page renders',() => {
    it ('populates the the login form', () => { 
        const location = {
            query: {
                business: {
                    category: "fast food",
                    id: 122,
                    location: "roysambu",
                    name: "minenetoo",
                    type:"cuisine",
                    user_id:1
                }
            }
        }
        expect(shallow(<EditBusinessForm location={location}/>)).toMatchSnapshot()
    });

} )