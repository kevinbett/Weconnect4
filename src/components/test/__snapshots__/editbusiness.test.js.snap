// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login page renders populates the the login form 1`] = `
<div>
  <Navigation />
  <div
    id="main-registration-container"
  >
    <div
      id="register"
    >
      <h3>
        Edit Business
      </h3>
      <form
        onSubmit={[Function]}
      >
        <p
          className="errorMsg"
        />
        <label>
          Name
        </label>
        <input
          name="name"
          onChange={[Function]}
          type="text"
          value="minenetoo"
        />
        <label>
          Type:
        </label>
        <input
          name="type"
          onChange={[Function]}
          type="text"
          value="cuisine"
        />
        <label>
          category
        </label>
        <input
          name="category"
          onChange={[Function]}
          type="text"
          value="fast food"
        />
        <label>
          Location
        </label>
        <input
          name="location"
          onChange={[Function]}
          type="text"
          value="roysambu"
        />
        <input
          className="button"
          type="submit"
          value="Edit Business"
        />
      </form>
    </div>
  </div>
</div>
`;
